settings.py

```diff
@@ -696,8 +696,8 @@ def get_env_map(name, default={}):
     'default': {
         'BACKEND': 'django.core.cache.backends.memcached.MemcachedCache',
         'LOCATION': '{0}:{1}'.format(
-            'cache',
-            '11211',
+            os.environ.get('MEMCACHED_HOST'),
+            os.environ.get('MEMCACHED_PORT'),
```

Comment:
You should provide the default values to get in case the environment is not set.

Reply:
well I looked at the other environment variables and i couldn't see that they had any defaults, so i just went ahead and did the samething here. Using `cache` as default is tightly coupled with using the docker-compose files that you provide. Is that what you want? or would you rather use localhost?

Reply:
Also PR #61 somewhat provides defaults at least when using docker-compose.

Reply:
Most of them do, the only ones that do not have them are the required ones (or introduced in the beginning). The problem I see is that somebody can updated docker image without updating the docker-compose (eg. because he is using custom one).

Anyway I've added the defaults in 7c819cc.